lib/themes/dosomething/paraneue_dosomething/templates/reportback/reportback-permalink.tpl.php

```diff
@@ -18,7 +18,7 @@
 <?php if ($is_current): ?>
   <?php echo $copy_vars['owners_rb_subtitle'] ?>
   <?php echo $copy_vars['owners_rb_scholarship'] ?>
-  <?php echo $reportback->caption ?>
+  <?php echo check_plain($reportback->caption) ?>
```

Comment:
This should be ok cos its being output on the interwebs. The change I made to the DS API module was to use `filter_xss` instead of `check_plain` because iOS was displaying the funky `&#39`'s